pkgs/development/python-modules/colorgram/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pillow
+}:
+
+buildPythonPackage rec {
+  pname = "colorgram.py";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname;
+    inherit version;
+    sha256 = "e77766a5f9de7207bdef8f1c22a702cbf09630eae3bc46a450b9d9f12a7bfdbf";
+  };
+
+  propagatedBuildInputs = [ pillow ];
+
```

Comment:
please add tests, you may need to check out from git hub

```
writing colorgram.py.egg-info/PKG-INFO
writing dependency_links to colorgram.py.egg-info/dependency_links.txt
writing requirements to colorgram.py.egg-info/requires.txt
writing top-level names to colorgram.py.egg-info/top_level.txt
reading manifest file 'colorgram.py.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'colorgram.py.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s
````

tests will ensure that your package isn't broken by version bumps